code/modules/clothing/gloves/_gloves.dm

```diff
@@ -33,7 +33,7 @@
 
 /obj/item/clothing/gloves/worn_overlays(mutable_appearance/standing, isinhands = FALSE)
 	. = ..()
-	if(!isinhands)
+	if(isinhands)
 		return
```

Comment:
seriously?

Reply:
yeah it was funny how it rendered blood on your hands when you held bloody gloves

Reply:
💀